Sprint-2/errors/1.js

```diff
@@ -4,10 +4,10 @@
 // Try playing computer with the example to work out what is going on
 
 function convertToPercentage(decimalNumber) {
-  const decimalNumber = 0.5;
+  decimalNumber = 0.5; // variable redeclaration
```

Comment:
You can see from the function 'signature' on line 6 that `convertToPercentage` is meant to accept one parameter (`decimalNumber`). But if the `decimalNumber` parameter is always overwritten like on line 7, then passing the parameter will never work.

Try to get this function to work by passing a value to it on line 13 (eg `convertToPercentage(0.1)`. What do you need to change or remove to make that work?

Reply:
remove line 7